cli/assets/CounterTemplate.t.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.13;
 
-import "forge-std/Test.sol";
-import "../src/Counter.sol";
+import {Test} from "forge-std/Test.sol";
```

Comment:
This is for the default contracts you get with `forge init` right? One consequence of this change is that `console2.log` is no longer in scope by default. I agree that named imports are preferable, but for better UX we may want to also import `console2` in both template files
```suggestion
import {Test, console2} from "forge-std/Test.sol";
```

Reply:
Sure! Adding `console2` make sense.

Is it better to add an example,
```
function testIncrement() public {
        counter.increment();
        console2.log(counter.increment());  --> example
        assertEq(counter.number(), 1);
 }
```
It can be commented.

Reply:
I'd leave out the console.log, we should still aim to keep these templates minimal